test/functional/page_objects/visualize_page.ts

```diff
@@ -113,8 +112,8 @@ export class VisualizePageObject extends FtrService {
     });
   }
 
-  public async clickRefresh() {
-    if (await this.visChart.isNewChartsLibraryEnabled()) {
+  public async clickRefresh(isNewLibrary = false) {
+    if ((await this.visChart.isNewChartsLibraryEnabled()) || isNewLibrary) {
```

Comment:
nit: since this is a general page component might be nice to say `isNewChartLibrary`.

```suggestion
  public async clickRefresh(isNewChartLibrary = false) {
    if ((await this.visChart.isNewChartsLibraryEnabled()) || isNewChartLibrary) {
```